lwgenerate/azure/azure.go

```diff
@@ -242,6 +256,11 @@ func (args *GenerateAzureTfConfigurationArgs) Generate() (string, error) {
 
 func createRequiredProviders() (*hclwrite.Block, error) {
 	return lwgenerate.CreateRequiredProviders(
+		lwgenerate.NewRequiredProvider(
+			"lacework",
+			lwgenerate.HclRequiredProviderWithSource(lwgenerate.LaceworkProviderSource),
+			lwgenerate.HclRequiredProviderWithVersion(lwgenerate.LaceworkProviderVersion),
```

Comment:
I wonder if we can bump this Lacework provider version. Latest is 0.26 https://github.com/lacework/terraform-provider-lacework/tags

Reply:
Do we also need to implement this for AWS/GCP?

Reply:
AWS and GCP already had the lacework provider section in place, this issue only affecting Azure when running `terraform plan/apply`